src/dwarf.rs

```diff
@@ -289,7 +286,7 @@ fn parse_debug_line_cu(
 
     Ok(DebugLineCU {
         prologue,
-        _standard_opcode_lengths: std_op_lengths,
+        _standard_opcode_lengths: std_op_lengths.to_vec(),
```

Comment:
Yep. The member is not used currently. I wanted to keep it around for the time being until I am sure it won't be needed down the line (I didn't introduce it). In the end it will boil down to the same decision as above: if we switch to `gimli` this code will end up being removed and that's it.